mods/action.js

```diff
@@ -187,7 +187,7 @@ ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
     apiRequest.body.action = defBody.action;
     apiRequest.body.format = apiRequest.body.format || defBody.format || 'json';
     apiRequest.body.formatversion = apiRequest.body.formatversion || defBody.formatversion || 1;
-    if (!apiRequest.body.hasOwnProperty('continue')) {
+    if (!apiRequest.body.hasOwnProperty('continue') && apiRequest.action === 'query') {
```

Comment:
Please note this change, it's not really related to the task, but we get a warning from the API when we add `continue` for non-query requests